manifest.toml

```diff
@@ -2,8 +2,8 @@ packaging_format = 2
 
 id = "peertube_remote_runner"
 name = "PeerTube Remote Runner"
-description.en = "Remote runner to offload high CPU/long job (such as transcoding) from Peertube main server"
```

Comment:
I'm against this change, the role of that app is already complicated enough, more explicit explanations are needed.

Reply:
The `DESCRIPTION.md` file is the place you are looking for to provide proper explanation.

Reply:
But it's not that long, people won't look at the description that easily, and this one doesn't convey a lot of meaning if you don't already know the purpose if this app :(
I feel like it's a step backward :/

Reply:
So I tried to find a middle ground by extending it a bit, but not much, and convey more meaning in the French version (which was clearly worse than the English one)